Vungle/VungleAdapter/ALVungleMediationAdapter.m

```diff
@@ -639,7 +640,7 @@ - (void)bannerAdDidFailToLoad:(VungleBanner * _Nonnull)banner withError:(NSError
 - (void)bannerAdDidPresent:(VungleBanner * _Nonnull)banner
 {
     NSString *creativeIdentifier = banner.creativeId;
-    if ( ALSdk.versionCode >= 6150000 && [creativeIdentifier al_isValidString] )
+    if ( ALSdk.versionCode >= kALSdkVersionNeeded && [creativeIdentifier al_isValidString] )
     {
         [self.adViewAdDelegate didDisplayAdViewAdWithExtraInfo: @{@"creative_id" : banner.creativeId}];
     }
```

Comment:
We might use `creativeIdentifier` instead of `banner.creativeId` here, like below.  Thanks. :)